settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -86,23 +86,20 @@ object TypelevelSettingsPlugin extends AutoPlugin {
       val warnings211 =
         Seq("-Ywarn-numeric-widen") // In 2.10 this produces a some strange spurious error
 
-      val warnings212 = Seq("-Xlint:-unused,_")
+      val warnings212 = Seq.empty[String]
 
-      val removed213 = Set("-Xlint:-unused,_", "-Xlint")
+      val removed213 = Set(
+        "-Xlint",
```

Comment:
Why do we remove `-Xlint` for 2.13?

Reply:
I don't believe that `-Xlint` on its own exists in 2.13 - I think it was replaced by the multi-argument one `-Xlint:_`, but I could be misremembering

Reply:
Hmm.. I tried briefly both `-Xlint` and `-Xlint:_` for both Scala 2.12 and 2.13. And seems all these combinations work in a similar way:
```
$ scala-cli compile -j 11 -S 2.12 '-Xlint' test1.scala 
[warn] ./test1.scala:2:31: parameter value ord in method hello is never used
[warn]   def hello[A](a: A)(implicit ord: Ordering[A]): Unit = {
[warn]                               ^
$ scala-cli compile -j 11 -S 2.12 '-Xlint:_' test1.scala 
[warn] ./test1.scala:2:31: parameter value ord in method hello is never used
[warn]   def hello[A](a: A)(implicit ord: Ordering[A]): Unit = {
[warn]                               ^
$ scala-cli compile -j 11 -S 2.13 '-Xlint' test1.scala 
[warn] ./test1.scala:2:31: parameter value ord in method hello is never used
[warn]   def hello[A](a: A)(implicit ord: Ordering[A]): Unit = {
[warn]                               ^^^^^^^^^^^^^^^^
$ scala-cli compile -j 11 -S 2.13 '-Xlint:_' test1.scala 
[warn] ./test1.scala:2:31: parameter value ord in method hello is never used
[warn]   def hello[A](a: A)(implicit ord: Ordering[A]): Unit = {
[warn]                               ^^^^^^^^^^^^^^^^
~
```
Although it is hard to say empirically whether they are fully interchangeable or not.

Reply:
Anyway, removing `-Xlint` there is no `-Xlint:_` added for 2.13.